### PR TITLE
fix issue #11, settings, select size, output folder.

### PR DESCRIPTION
**Pull request by GeorgLegato**

### CODE REVIEW

File: .gitignore
Comment (v8hid) [inline .gitignore:1]: Should be removed because the output already changed.

**Review by v8hid — CHANGES_REQUESTED** [review]
1. remove .gitignore update
2. set the default infzoom_outpath to [script_path]/output/infinite-zoom
3. Set mask_with dynamically
Thank you it's great <3

**Review by v8hid — APPROVED** [review]

**Comment by v8hid** [comment]
Great job @GeorgLegato, 
I think it would be better to set the `default infzoom_outpath` to `[script_path]/output/infinite-zoom`
What do you think?

**Comment by GeorgLegato** [comment]
gitignore rollbacked, mask_width is now outputsize/4, works well.... but:
when setting outputsize to 610 video is still 512... checking render and video settings

**Comment by v8hid** [comment]
> gitignore rollbacked, mask_width is now outputsize/4, works well.... but: when setting outputsize to 610 video is still 512... checking render and video settings

it might be because of Gradio video component have static size of 512 * 512

**Pull request merged**